tensorboard/webapp/routes/dashboard_deeplink_provider.ts

```diff
@@ -45,7 +46,12 @@ const COLOR_GROUP_REGEX_VALUE_PREFIX = 'regex:';
  * Provides deeplinking for the core dashboards page.
  */
 @Injectable()
-export class DashboardDeepLinkProvider extends DeepLinkProvider {
+export class DashboardDeepLinkProvider<
+  T extends FeatureFlags = FeatureFlags
```

Comment:
what does this do for you? The idea is to be able to pass a new FeatureFlag type that has extra flags in TB Corp? Then update FeatureFlags in OSS afterwards?

Reply:
This says that it needs to be an object that contains all the keys of OSS FeatureFlags but can also have more (i.e. TB Corp feature flags)